engine/execution/state/bootstrap/bootstrap_test.go

```diff
@@ -47,7 +47,7 @@ func TestBootstrapLedger(t *testing.T) {
 }
 
 func TestBootstrapLedger_ZeroTokenSupply(t *testing.T) {
-	expectedStateCommitmentBytes, _ := hex.DecodeString("f04b063fa45655110ff10ee3153726366966efc8ebcdc730d27b9971eca5cedf")
+	expectedStateCommitmentBytes, _ := hex.DecodeString("e4b748da1215452fcaa60f5549d261bf3f76a8a6e9a8388a6d8ee73193b4e331")
```

Comment:
hmm, is this expected? did the execution state changed because of the way keys are encoded?

Reply:
Good question, that seems to be caused by a core-contract update (domain tag is updated on a contract and this PR uses the new core-contract repo version). My guess is that since the contract is deployed to the service account, the state commitment has changed. Do you think that's a valid reason?